package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.14.1",
   "description": "Video-React is a web video player built from the ground up for an HTML5 world using React library.",
   "main": "lib/index.js",
+  "module": "./dist/video-react.es.js",
```

Comment:
Can you elaborate on why you're doing this? Is there something in the vite docs / rollup docs indicating that this is the way to go? I'm looking here https://stackoverflow.com/a/42817320 which seems to indicate that `module` is not standard.

Reply:
ah, yea having this set to module here means we dont need to solve the "named export" problem we ran into with rollup and hit again with vite. our current approach works with _old_ versions of rollup but i dont think it does with the newest version and vite has said people who dont do something similar are bad and they should [push them into the future](https://github.com/vitejs/vite/issues/162#issuecomment-630573458)

Reply:
i wanted to change as little as possible really... now i'm trying to update package.json to use the exports pattern instead but running into some problems